sql/share/errmsg-utf8.txt

```diff
@@ -7132,6 +7132,8 @@ ER_CANNOT_CONVERT_CHARACTER
         eng "Cannot convert '%s' character 0x%-.64s to '%s'"
 ER_INVALID_DEFAULT_VALUE_FOR_FIELD  22007
         eng "Incorrect default value '%-.128s' for column '%.192s'"
+ER_SYSTEMD_LISTEN_FDS
+        eng "Systemd error for configured ListenStream '%.192s'"
```

Comment:
We try to keep errmsg-utf8.txt backward compatible (not sure what were the reasons though).

All in all it means every new error message must appear last in main branch.